source/_docs/troubleshooting_general.markdown

```diff
@@ -0,0 +1,43 @@
+---
+title: "General troubleshooting"
+description: "General troubleshooting information"
+---
+
+This page provides some information about more generic troubleshooting topics.
+
+## Home Assistant went into recovery mode
+
+### Symptom: Home Assistant is in recovery mode
+
+On top of the page you see a red banner. On the **Overview** page, you see a **Recovery mode** notification.
+
+![image](/images/docs/troubleshooting/recovery_mode_active.png)
```

Comment:
It looks like there's a frontend problem with retrieving the logs in the example image. Is that intended? It's a bit confusing, I think.